ansible/munin/list_sqs_queues.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+import boto3
+import os
+
+# ref: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Client.get_queue_attributes
+num_of_msgs_attr = 'ApproximateNumberOfMessages'
+
+environment = os.getenv('ENV', 'staging')
```

Comment:
I guess this should be a host variable filled in from ansible?

Reply:
You're right, makes more sense to have it filled by ansible.